lib/hanami/routing/recognized_route.rb

```diff
@@ -140,6 +140,7 @@ def routable?
       #
       # @see Hanami::Routing::Endpoint
       def destination
+        return '' unless @endpoint
         @destination ||= begin
```

Comment:
what do you think about catching exception instead? since we have already got `begin` block here, you can always do `rescue NoMethodError` then return a value